sbi/inference/base.py

```diff
@@ -253,7 +253,10 @@ def _run_simulations(
             prior_mask: Whether each simulation came from a prior parameter sample.
         """
 
-        if proposal is None:
+        # Some proposals (e.g. flows) can not deal with `num_sims=0`. So we just draw
+        # 0 prior samples instead (does not matter because it is 0 samples anyway).
```

Comment:
maybe write here that this works because pytorch distributions do accept 0 samples.

Reply:
I feel like the more correct thing to say here would be "because the prior accepts 0 samples" (need not be a torch dist). And I think this is a bit redundant / clear anyway.